src/pkg/backup/details/details.go

```diff
@@ -395,7 +402,11 @@ func (i OneDriveInfo) Headers() []string {
 // out to a terminal in a columnar display.
 func (i OneDriveInfo) Values() []string {
 	return []string{
-		i.ItemName, i.ParentPath, humanize.Bytes(uint64(i.Size)), i.Owner,
-		common.FormatTabularDisplayTime(i.Created), common.FormatTabularDisplayTime(i.Modified),
+		i.ItemName,
+		i.ParentPath,
+		humanize.Bytes(uint64(i.Size)),
+		i.Owner,
```

Comment:
Does this line up with the `i.WebURL` in the previous section (L373)?

Reply:
webURL is just a more readable version of the owner (which should be the siteID).